database/database_test.go

```diff
@@ -239,6 +275,49 @@ func TestStatesWithStatus(t *testing.T) {
 	}
 }
 
+func TestSetAutomatedPendingRemoval(t *testing.T) {
+	resetDB()
+
+	testStates := []DomainState{
+		{Name: "preloaded.test", Status: StatusPreloaded},
+		{Name: "pending.test", Status: StatusPending},
+		{Name: "pendingremoval.test", Status: StatusPendingRemoval},
+		{Name: "rejected.test", Status: StatusRejected},
+		{Name: "removed.test", Status: StatusRemoved},
+		{Name: "unknown.test", Status: StatusUnknown},
+		{Name: "pendingautomatedremoval.test", Status: StatusPendingAutomatedRemoval}}
+
+	for _, state := range testStates {
+		if err := testDB.PutState(state); err != nil {
+			t.Errorf("cannot put state %s", err)
```

Comment:
This should also be a t.Fatalf (since you are also not intending to test PutState here).

Reply:
Done.